src/Microsoft.Health.Fhir.Core/Features/Operations/Export/ExportJobTask.cs

```diff
@@ -677,7 +677,19 @@ private async Task<SearchResult> GetGroupPatients(string groupId, List<Tuple<str
         {
             if (!queryParametersList.Exists((Tuple<string, string> parameter) => parameter.Item1 == KnownQueryParameterNames.Id || parameter.Item1 == KnownQueryParameterNames.ContinuationToken))
             {
-                var patientIds = await _groupMemberExtractor.GetGroupPatientIds(groupId, groupMembershipTime, cancellationToken);
+                HashSet<string> patientIds = await _groupMemberExtractor.GetGroupPatientIds(groupId, groupMembershipTime, cancellationToken);
+
+                if (patientIds.Count == 0)
+                {
+                    _logger.LogInformation($"Group {groupId} does not have any patient ids as members.");
+                    SearchResult emptySearchResult = new SearchResult(
+                        results: new List<SearchResultEntry>(),
+                        continuationToken: null,
+                        sortOrder: null,
+                        unsupportedSearchParameters: new List<Tuple<string, string>>());
```

Comment:
nit: why not: `new SearchResult(0, new List<Tuple<string, string>>())`?

Reply:
Depends on how you feel, I would even step further and create
```
public static SearchResult Empty(IReadOnlyList<Tuple<string, string>> unsupportedSearchParameters=null)=>
new SearchResult(0, unsupportedSearchParameters?? new List<Tuple<string,string>>()
```
method inside search result class and use it among other places in code.  (I can see few cases where we create empty search result other than this part of code like `CreateEmptySearchResponse` method  or tests in SearchServiceTests.)

Reply:
Added an extension and updated code to use the new `SearchResult.Empty()` method.